virttest/data_dir.py

```diff
@@ -201,6 +201,10 @@ def get_deps_dir(target=None):
 def get_tmp_dir():
     return data_dir.get_tmp_dir()
 
+def get_public_tmp_dir():
+    tmp_dir = data_dir.get_tmp_dir()
+    os.chmod(tmp_dir, 0755)
+    return tmp_dir
 
```

Comment:
I have read your discussion.
I suggest you that it's better to add bool param named public to give user permission or not.
The default of public maybe False.